packages/docs/src/basics/paragraph.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ICustomTable, IParagraph } from '@univerjs/core';
+
+export function isParagraphInTable(paragraph: IParagraph, tables: ICustomTable[]) {
```

Comment:
```suggestion
export function hasParagraphInTable(paragraph: IParagraph, tables: ICustomTable[]) {
```

Reply:
`isXXX` is usually considered as a property  or an variable instead of a function.